wsmux/mux.go

```diff
@@ -0,0 +1,26 @@
+package wsmux
+
+import (
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// Config values for Session
+type Config struct {
+	KeepAliveInterval    time.Duration
+	StreamAcceptDeadline time.Duration
+	RemoteCloseCallback  func()
+	Log                  Logger
```

Comment:
Can't we document the default value of all of these...
Ie. if I use `Config{}`, then `StreamAcceptDeadline == 0` and then presumably we set it to some sane default...

Let's document that default... like:
```
StreamAcceptDeadline time.Duration  // Defaults to 30 seconds
RemoteCloseCallback  func()  // Optional
Log                  Logger  // Optional
StreamBufferSize     int // 32kb
``` 
We can do that for all of these... Since they all have some default value...
Note: Values entered here was just illustrative...